wandb/sdk/internal/internal_api.py

```diff
@@ -157,12 +157,15 @@ def execute(self, *args, **kwargs):
         try:
             return self.client.execute(*args, **kwargs)
         except requests.exceptions.HTTPError as err:
-            res = err.response
-            logger.error("%s response executing GraphQL." % res.status_code)
-            logger.error(res.text)
-            self.display_gorilla_error_if_found(res)
+            self.log_http_error(err)
             raise
 
+    def log_http_error(self, err: requests.exceptions.HTTPError) -> None:
+        res = err.response
+        logger.error("%s response executing GraphQL." % res.status_code)
```

Comment:
```suggestion
        logger.error(f"{res.status_code} response executing GraphQL.")
```
total nit... just moving to py3 syntax

Reply:
yesssssss